index.html

```diff
@@ -8,25 +8,62 @@
     <title>Awesome Books</title>
 </head>
 <body>
+	<header>
+		<div class = "nav_container">
+			<a>
+				<h2>Awesome books</h2>
+			</a>
+			<div class="menu">
+				<ul>
+					<li><a class="nav-link" href="#bookList">Book List</a><hr/></li>
+					<li><a class="nav-link" href="#addBook">Add Book</a><hr/></li>
+					<li><a class="nav-link" href="#contact">Contact</a></li>
+				</ul>
+			</div>
```

Comment:
- [ ] As it is a requirement for this project, make sure that when you click on any of the links , the main area changes to display `only one of the three content sections`, all your three section are being displayed at the same time, please kindly fix that. Only one of the content sections is supposed to be displayed at a given time i.e. either 
    `Books list` or
    `Add book form` or
    `Contact info`.